tiled/serialization/table.py

```diff
@@ -49,14 +49,19 @@ def serialize_parquet(df, metadata, preserve_index=True):
     return memoryview(sink.getvalue())
 
 
-def serialize_csv(df, metadata, preserve_index=False):
+def serialize_csv(mimetype, df, metadata, preserve_index=False):
     file = io.StringIO()
+    if ";" in mimetype:
```

Comment:
As agreed, _this_ PR will only add one example of handling mimetype parameters. But we expect future PRs to handle more examples of this, so it may be worth generalizing to a separate function that we can reuse. That will also enable us to test it separately for correctness.

````py
def parse_mimetype(mimetype: str): -> tuple[str, dict]
    "Given a mimetype like 'x/y;a=b;c=d' return ('x/y', {'a': 'b', 'c': 'd'}."
    ...
```